doc/en/example/simple.rst

```diff
@@ -302,34 +302,30 @@ running from a test you can do something like this:
 
 .. code-block:: python
 
-    # content of conftest.py
+    # content of your_module.py
 
 
-    def pytest_configure(config):
-        import sys
+    _called_from_test = False
 
-        sys._called_from_test = True
 
+    # content of conftest.py
 
-    def pytest_unconfigure(config):
-        import sys
 
-        del sys._called_from_test
+    def pytest_configure(config):
+        your_module._called_from_test = True
 
-and then check for the ``sys._called_from_test`` flag:
+and then check for the ``your_module._called_from_test`` flag:
 
 .. code-block:: python
 
-    if hasattr(sys, "_called_from_test"):
+    if your_module._called_from_test:
         # called from within a test run
         ...
     else:
         # called "normally"
         ...
 
```

Comment:
i noted that we haven't yet a actual example execution block, so this example isn't actually executed demoing the effect -

Reply:
How do I do that?

Reply:
TBH I think the current example is fine, it is a really contrived use case anyways. 🤷‍♂